slack_wrapper/api.py

```diff
@@ -0,0 +1,48 @@
+import requests, json
+
+class SlackAPI():
+	def __init__(self, token):
+		self.TOKEN = token
+		self.domain = 'https://slack.com/api/'
+
+	def has_error(self, response_dict, log="An error ocurred:"):
+		if response_dict['ok']:
+			return False
+		else:
+			print(log)
+			print(response_dict['error'])
+			return True
+	
+	def api_method(self, method, extra_params={}):
+		"""executes any method and returns a dictionary"""
+		params = {'token': self.TOKEN}
+		params.update(extra_params)
+		response_dict = json.loads(requests.get(
+                                        self.domain + 'channels.list',
+                                        params=params
+                                ).text)
```

Comment:
@raylu simple: I didn't know that the second option existed and did the same, I'll change that